lib/main_common.pm

```diff
@@ -1705,7 +1705,6 @@ sub load_extra_tests_sdk {
 
 sub load_extra_tests_docker {
     my ($image_names, $stable_names) = get_suse_container_urls();
```

Comment:
I don't see any use of this variables at all , what I am missing ? 
( obviously not directly related to this PR , just thinking loud )

Reply:
Following the comment of @pdostal, we are now using those to skip the `docker_image` test, if `@$image_names` is empty.